src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryService.scala

```diff
@@ -133,6 +149,18 @@ class LibraryService (protected val argUserInfo: UserInfo,
     }
   }
 
+  def getAttributes(ns: String, name: String): Future[PerRequestMessage] = {
```

Comment:
It's somewhat confusing that this response is different than the `attributes` key of the `updateAttributes` response - because we're limiting to attributes where`k.namespace == AttributeName.libraryNamespace`.

Is it correct to say that the predicate above defines the set of attributes we call metadata? I've gotten a bit lost in the naming.